io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx

```diff
@@ -16,31 +16,27 @@ export function FormButtons(props: {
   return (
     <React.Fragment>
       <Grid
-        sx={
-          useSmallDevice()
-            ? {
-                flexGrow: 1,
-                position: "fixed",
-                zIndex: 1000,
-                bottom: 0,
-                left: 0,
-                padding: "1rem",
-                boxShadow: "0 0.5rem 1rem rgb(0 0 0 / 15%)",
-                backgroundColor: "#fff",
-              }
-            : { flexGrow: 1, marginTop: "48px" }
-        }
+        sx={{
+          position: { xs: "fixed", sm: "relative" },
+          zIndex: { xs: 1000, sm: 0 },
+          bottom: { xs: 0 },
+          left: { xs: 0 },
+          p: { xs: "1rem", sm: 0 },
+          boxShadow: { xs: "0 0.5rem 1rem rgb(0 0 0 / 15%)", sm: 0 },
+          bgcolor: { xs: "background.default" },
+          mt: { sm: 6 },
+        }}
         justifyContent="center"
         flexDirection="row"
         alignItems="center"
         container
         spacing={2}
       >
         <Grid
-          xs={3}
-          md={3}
-          lg={3}
-          xl={3}
+          xs={4}
+          md={4}
```

Comment:
Are you sure that it's necessary set also md-lg-xl ? MUI is mobile first, you could set only the "base value" if it's the same for all viewports